moped-database/migrations/1665693611607_add_component_geometry_tables/up.sql

```diff
@@ -0,0 +1,168 @@
+update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_intersections'  ) where component_name = 'Pavement Marking' and component_subtype = 'Sharrows (Wide Curb Lane)';
+update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_intersections'  ) where component_name = 'Signage' and component_subtype is null;
+update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_intersections'  ) where component_name = 'Crosswalk' and component_subtype = 'Continental';
+update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_intersections'  ) where component_name = 'Crosswalk' and component_subtype = 'Creative';
+update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_intersections'  ) where component_name = 'Crosswalk' and component_subtype = 'Raised';
+update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_street_segments') where component_name = 'Sidewalk' and component_subtype = 'Ramps';
+update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_street_segments') where component_name = 'Lighting' and component_subtype = 'Street lighting';
+update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_intersections'  ) where component_name = 'Lighting' and component_subtype = 'Intersection lighting';
+update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_intersections'  ) where component_name = 'Transit' and component_subtype = 'Bus stop';
+update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_street_segments') where component_name = 'Project Extent - Generic' and component_subtype is null;
+update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_intersections'  ) where component_name = 'Signage' and component_subtype = 'RRFB';
+update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_intersections'  ) where component_name = 'Intersection' and component_subtype = 'Roundabout';
+
+alter table moped_components alter column feature_layer_id set not null;
+
+alter table moped_proj_components
+  add column feature_layer_id_override integer references feature_layers(id);
+
+create table features (
+    id serial primary key, 
+    name character varying
+    );
+
+create table feature_signals (
+    signal_id integer default null,
+    knack_id character varying,
+    project_extent_id uuid,
```

Comment:
Re this conversation: https://austininnovation.slack.com/archives/CNUEPKLB1/p1666042882917659

The values that come over in the `project_extent_id` field appear to be UUID fields. My understanding from John & Mike is that these IDs have been used as unique identifiers for drawable layers. Is this a suitable solution?

Reply:
i missed this — yes!